m3u8/writer.go

```diff
@@ -689,6 +693,11 @@ func (p *MediaPlaylist) Encode() *bytes.Buffer {
 			p.buf.WriteString("VOD\n")
 		}
 	}
+	if p.PartTargetDuration > 0 {
+		p.buf.WriteString("#EXT-X-PART-INF:PART-TARGET=")
+		p.buf.WriteString(strconv.FormatFloat(float64(p.PartTargetDuration), 'f', 6, 64))
+		p.buf.WriteRune('\n')
```

Comment:
I think we should use consistent accuracy in the durations. At present, it is 3 digits (ms) everywhere else. But, as a next step, I'd like to have a parameter "durationDecimals"  or similar in the media playlist that you can set to 3, 6 or -1.